packages/source-iotsitewise/src/time-series-data/client/batchGetAggregatedPropertyDataPoints.ts

```diff
@@ -50,13 +51,15 @@ const sendRequest = ({
   // callback cache makes it convenient to capture request data in a closure.
   // the cache exposes methods that only require batch response entry as an argument.
   const callbackCache: BatchEntryCallbackCache = {};
-
   const batchSize = calculateNextBatchSize({ maxResults, dataPointsFetched });
 
+  // filter out duplicate requests
+  const uniqueRequests = uniqBy(batch, (entry) => entry.requestInformation);
```

Comment:
i think we may also want to omit a couple fields, such as `refId`, `cacheSettings`